ExRouter.js

```diff
@@ -30,6 +30,7 @@ export class ExRouteAdapter {
         }
         this.route = route;
         this.name = route.name;
+        if (props.title) this.route.title = props.title;
```

Comment:
Feels like I'm correcting something that should be handled earlier on in some other module, but this solves the issue with getting the parent route title.